// Copyright (c) 2020 Boston Dynamics, Inc.  All rights reserved.
//
// Downloading, reproducing, distributing or otherwise using the SDK Software
// is subject to the terms and conditions of the Boston Dynamics Software
// Development Kit License (20191101-BDSDK-SL).

// LogAnnotationService allows adding information to the robot's log files.
syntax = "proto3";

package bosdyn.api;

option java_outer_classname = "LogAnnotationServiceProto";

import "bosdyn/api/log_annotation.proto";

// This service is a mechanism for adding information to the robot's log files.
service LogAnnotationService {
    // Add the specified information to the robot's log files.
    rpc AddLogAnnotation(AddLogAnnotationRequest) returns (AddLogAnnotationResponse) {}
}


